weave/flow/scorers/__init__.py

```diff
@@ -0,0 +1,49 @@
+from weave.flow.scorers.base_scorer import (
```

Comment:
each of these files have their own dependencies.  If import fails, we should tell the user to install the relevant extra instead of just erroring